src/module/chat-message/listeners/cards.ts

```diff
@@ -71,16 +71,17 @@ export const ChatCards = {
                 else if (action === "spellCounteract") spell?.rollCounteract(event);
                 else if (action === "spellTemplate") spell?.placeTemplate();
                 else if (action === "selectVariant") {
-                    const spellLvl = Number($html.find<HTMLDivElement>("div.chat-card").attr("data-spell-lvl")) || 1;
+                    const slotLevel =
+                        Number($html[0].querySelector<HTMLElement>("div.chat-card")?.dataset.slotLvl) || 1;
                     const overlayIdString = $button.attr("data-overlay-ids");
                     const originalId = $button.attr("data-original-id") ?? "";
                     if (overlayIdString) {
                         const overlayIds = overlayIdString.split(",").map((id) => id.trim());
-                        const variantSpell = spell?.loadVariant({ overlayIds, castLevel: spellLvl });
+                        const variantSpell = spell?.loadVariant({ overlayIds, castLevel: slotLevel });
```

Comment:
Only 90% sure on this but shouldn't the chat message listeners for loading a spell always use the cast level?

Reply:
@In3luki Do you remember what this was expecting?

Reply:
That should be the cast level. `loadVariant` applies fixed heigthening based on that value.